distribution/docs/src/main/resources/_contents/_broker/configure-ha.adoc

```diff
@@ -0,0 +1,78 @@
+The following sections describe how to configure two ${branding} broker installations to create a highly available cluster.
```

Comment:
Are these broker documentation files being included anywhere in the full documentation?